x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/assignment/AssignmentStats.java

```diff
@@ -274,6 +304,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             if (avgInferenceTimeLastPeriod != null) {
                 builder.field("average_inference_time_ms_last_minute", avgInferenceTimeLastPeriod);
             }
+            builder.field("inference_cache_hit_count_last_minute", cacheHitCountLastPeriod);
```

Comment:
It seems weird to me to include this stat when `cacheHitCount == null`. It seems like either we include both when they are missing or neither.

Reply:
You're right it's weird, but I made it match `throughput_last_minute` which it logically goes with.  Higher up the file `inference_count` is omitted when nothing has happened, but immediately above `throughput_last_minute` isn't.